### ISSUE TITLE
Module require error in version 7.26.1

### ISSUE THREAD

**Issue by albertstill** [testing-library/dom-testing-library#793]
Node 12.18.4 and @testing-library/dom 7.26.1, pinning to a lower version of @testing-library/dom solved the issue for me. I believe this was the call [site](https://github.com/testing-library/dom-testing-library/blob/bc65b91fed81d900e8eb2aff1813752a4f487869/src/get-node-text.js#L1).

Require was triggered by Cypress not sure if this has anything to do with it though.

```
Oops...we found an error preparing this test file:

  support/index.js

The error was:

Error: Webpack Compilation Error
/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/dom.esm.js
Module not found: Error: Can't resolve './helpers' in '/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library'
Looked for and couldn't find the file at the following paths:
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.js]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.json]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.jsx]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.mjs]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.coffee]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.ts]
[/Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/helpers.tsx]
 @ /Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/dom/dist/@testing-library/dom.esm.js 387:15-35
 @ /Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/cypress/dist/index.js
 @ /Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/cypress/dist/add-commands.js
 @ /Users/albert.still/Documents/GitHub/fe-server-search-listings/node_modules/@testing-library/cypress/add-commands.js
 @ ./support/index.js

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies

Fix the error in your code and re-run your tests.
```

**Comment by datdoan** [comment]
I have the same issue but different package/version

**Not working**
@testing-library/cypress@7.0.1
Cypress@5.0.0, Cypress@5.2.0, Cypress@5.3.0

**Working**
@testing-library/cypress@7.0.1
Cypress@4.3.0

**Comment by johncalvinroberts** [comment]
Same issue here

**Comment by eps1lon** [comment]
Fixed in 53b547f016b673a3299c9782286f8e3da53a95b3. 

Accidentally committed to `master` directly. Had different defaults in the GitHub UI than I'm used to.

**Comment by johncalvinroberts** [comment]
@eps1lon which version will this be published in?
